src/benchmarks/micro/libraries/System.Text.Json/Utf8JsonReader/Perf.Get.cs

```diff
@@ -23,214 +25,189 @@ private static byte[] GetJsonBytes<T>(T elem)
             return Encoding.UTF8.GetBytes(elem.ToString());
         }
 
-        [Benchmark]
-        public byte GetByte()
+        [Benchmark(OperationsPerInvoke = OperationsPerBenchmark)]
```

Comment:
What is the benefit of using `OperationsPerInvoke` here?

Reply:
@layomia https://github.com/dotnet/performance/blob/main/docs/microbenchmark-design-guidelines.md#operationsperinvoke